doc/nrf/ug_nrf_cloud.rst

```diff
@@ -56,6 +56,54 @@ MQTT overview
 * Once connected, the device subscribes to the desired MQTT topics.
 * Each MQTT publish event contains the MQTT topic and the payload.
 
+Security
+********
+
+A device can successfully connect to `nRF Cloud`_ using REST if:
+
+* the device contains a correct x509 CA certificate, and private key
+* the public key derived from the private key is registered with an nRF Cloud account
+* the device calls nRF Cloud REST APIs which require a JSON Web Token (JWT) by providing a JWT signed by the private key
+
+A device can successfully connect to `nRF Cloud`_ using MQTT if:
+
+* the device contains a correct x509 CA certificate, device certificate, and private key
+* the device id and device certificate are provisioned with nRF Cloud
+* the device id is associated with an nRF Cloud account
+
+`nRF Cloud`_ supports two different ways for creating and installing these certificates both in the device and the cloud.
+See `nRF Cloud Provisioning`_ for more information.
+
+* Online
```

Comment:
Is "Online" provisioning used anywhere else?  Why not just say this is JITP provisioning?